### DISCUSSION TITLE
Inject different html tags per route

### DISCUSSION THREAD

**Discussion by marinsokol5** [facebook/docusaurus#9075] [General]
Hey, 

My use case is that I want different CSP rules per different route, mostly because I have a home page which works with loose CSP rules to accomodate for some of the Docusaurus stuff, but I also want to have a more stripped page which should abide by stricter CSP rules.

Currently I use [injectHtmlTags](https://docusaurus.io/docs/2.2.0/api/plugin-methods/lifecycle-apis#injectHtmlTags) API for this, but is it in anyway possible to have different HTML tags returned for different routes?

**Comment by slorber** [comment]
You can use the `<Head>` component for individual page metadata:
https://docusaurus.io/fr/docs/seo#single-page-metadata

```jsx
<Head>
  <meta http-equiv="Content-Security-Policy" content="..." />
</Head>
```
